loginApp/login.app.js

```diff
@@ -0,0 +1,99 @@
+function loadModule() {
+    (function () {
+        'use strict';
+
+        angular.module('chaise.loginapp', [
+            'chaise.alerts',
+            'chaise.authen',
+            'chaise.login',
+            'chaise.modal',
+            'chaise.utils',
+            'ermrestjs',
+            'ngCookies',
+            'ngAnimate',
+            'ui.bootstrap'])
+
+            .config(['$cookiesProvider', function ($cookiesProvider) {
```

Comment:
The only nitpick I have is that there is no need to have 3 separate configuration blocks.
```js
.config(['$cookiesProvider', '$uibTooltipProvider', '$logProvider', function ($cookiesProvider, $uibTooltipProvider, $logProvider) {
   $cookiesProvider.defaults.path = '/';
   $uibTooltipProvider.options({ appendToBody: true });
   $logProvider.debugEnabled(chaiseConfig.debug === true);
}])
```